server/flaskr/database/handlers/meeting_data_handler.py

```
@@ -201,6 +201,19 @@ def get_next_meeting(cls, currentMeetingId: int):
             return None
         return cls.get_meeting_agenda(nextId)
 
+    @classmethod
+    def get_meeting_users(cls, meetingId):
```

Ruff (0.15.20)

[warning] Missing return type annotation for classmethod `get_meeting_users`

(ANN206)